cypress/support/fragments/circulation/newNoticePolicyTemplate.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+import getRandomPostfix from '../../utils/stringTools';
+import { Button, TextField, TextArea, KeyValue, Checkbox, Link, Heading } from '../../../../interactors';
+import richTextEditor from '../../../../interactors/rich-text-editor';
+import { NOTICE_CATEGORIES } from './notice-policy';
+
+const newButton = Button({ id: 'clickable-create-entry' });
+const actionsButton = Button('Actions');
+const tokenButton = Button('{ }');
+const addTokenButton = Button('Add token');
+const nameField = TextField({ id: 'input-patron-notice-name' });
+const subjectField = TextField({ id: 'input-patron-notice-subject' });
+const descriptionField = TextArea({ id: 'input-patron-notice-description' });
+const bodyField = richTextEditor();
+const defaultUi = {
+  name: `Test_template_${getRandomPostfix()}`,
+  description: 'Template created by autotest team',
+  subject: 'Subject_Test',
+  body: 'Test_email_body',
+};
+
+export default {
+  defaultUi,
+  waitLoading() {
+    cy.do(Link('Patron notice templates').click());
+    cy.expect(Heading('Patron notice templates').exists());
+  },
+  startAdding() {
+    cy.do(newButton.click());
+  },
+  create(noticePolicyTemplate) {
+    this.checkForm();
+    this.fillGeneralInformation(noticePolicyTemplate);
+  },
+  checkNewButton() {
+    cy.expect([
+      newButton.exists(),
+      cy.get('[id="clickable-create-entry"]').should('not.be.disabled')
+    ]);
+  },
+  save() {
+    cy.do([
+      Button({ id: 'footer-save-entity' }).click(),
+      Button({ icon: 'times' }).click(),
+    ]);
+  },
+  checkForm() {
+    cy.expect([
+      Heading('New patron notice template').exists(),
```

Comment:
add implementation of "Pay attention on Category" - "Loan category selected by default"